cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -322,9 +322,6 @@ def certificate_activation_handler(request, course_key_string):
     POST
         json: is_active. update the activation state of certificate
     """
-    # Only global staff (PMs) are able to activate/deactivate certificate configuration
```

Comment:
Sweet! I imagine we'd want a formal thumbs / two of them so I'll keep looking and see if anybody else wants to test this. (The sandbox still works as well if you login as honor@example.com) for testing)